R/lazyframe__lazy.R

```diff
@@ -926,6 +982,37 @@ LazyFrame_dtypes = method_as_property(function() {
     unwrap("in $dtypes()")
 })
 
+#' @title Collect and profile a lazy query.
+#' @description This will run the query and return a tuple containing the materialized DataFrame and
```

Comment:
```suggestion
#' @description This will run the query and return a list containing the materialized DataFrame and
```
I guess an R list is the equivalent of Python tuple?